pkg/mount/mountinfo_darwin.go

```diff
@@ -0,0 +1,8 @@
+// build +darwin
+
+package mount // import "github.com/docker/docker/pkg/mount"
+
+func parseMountTable(f FilterFunc) ([]*Info, error) {
+	// Do NOT return an error!
+	return nil, nil
```

Comment:
I don't understand why we need this darwin-specific `parseMountTable`.  why we can use that function in mountinfo_unsupported.go ?